src/sysext/sysext.c

```diff
@@ -76,6 +77,8 @@ static MutableMode arg_mutable = MUTABLE_NO;
 /* Is set to IMAGE_CONFEXT when systemd is called with the confext functionality instead of the default */
 static ImageClass arg_image_class = IMAGE_SYSEXT;
 
+static const char *mutable_extensions_base_dir = "/var/lib/extensions.mutable";
```

Comment:
hmm, we don't really like global variables in systemd. There are some exceptions, but generally the rule is that only stuff parsed from cmdline, env vars and suchlike (i.e. stuff that is inherently per-process in its origin) should be placed in global variables, and then they should get an "arg_" prefix, to indicate what they are: "arguments" to the program.

THis one here doesn't really qualify, does it?

Reply:
also, i don't grok the field? it's only ever set to a fixed value, and never changed? why is this a variable at all?

Reply:
shouldn't this just be part of the image_class_info[] data?

Reply:
Should have been a `#DEFINE` instead, just didn't want to repeat the string. About being in image_class_info - now I got confused. You pointing it out made sense to me, so I just made it a part of the image_class_info. But the I noticed that the sysext manpage and the spec (https://github.com/uapi-group/specifications/pull/78) use `/var/lib/extensions.mutable/` for both sysexts and confexts. So, which way we go?